src/components/Content/Content.jsx

```diff
@@ -0,0 +1,76 @@
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import { getImage } from "../../services/nasa";
+
+import DateField from "../DateField";
+import Description from "../Description";
+import Image from "../Image";
+
+const ERROR_MESSAGE = "There was an error, please try again.";
+
+const Content = () => {
+	const [imageOfTheDay, setImageOfTheDay] = useState({});
+	const todaysDate = new Date().toISOString().split("T")[0];
+	const [date, setDate] = useState(todaysDate);
+	const [error, setError] = useState(false);
+	const [loading, setLoading] = useState(false);
+	const [errorMsg, setErrorMsg] = useState(ERROR_MESSAGE);
+
+	const fetchImage = async (date) => {
```

Comment:
Good Job, but i's a best practice to have fetching logic in a custom hook, so the components are more `stateless` and separate responsibility. Check these examples.
[Article How to fetch data with React Hooks](https://www.robinwieruch.de/react-hooks-fetch-data/)
[CodeSandbox useFetch](https://codesandbox.io/s/usereducer-example-71pbj8?file=/src/App.js)